### DISCUSSION TITLE
Computing Power for Chaste

### DISCUSSION THREAD

**Discussion by chinmaykh** [Chaste/Chaste#190] [Q&A] [answered]
I'm thinking of using Chaste for agent-based tissue modelling. I want to know what kind of computing power Chaste demands. Does it run efficiently on GPUs? Does it require a lotta RAM?

All inputs would be appreciated. I really need to decide on a computer quickly.

**Comment by jmpf** [comment, marked as answer]
Agent-based tissue modelling does not use any GPU (yet).  There is an on-going project to implement GPU cell-based models but that's at least a year away from release.

Most agent-based models use sequential (1-core) CPU.  The only exception is the over-lapping spheres model and details of its parallelisation are at https://chaste.cs.ox.ac.uk/trac/wiki/PaperTutorials/Harvey2015

As to the amount of RAM: how big is your tissue?
